src/ibmca_rsa.c

```diff
@@ -300,10 +301,13 @@ static int ibmca_rsa_init(RSA * rsa)
     return 1;
 }
 
+static int (*ibmca_rsa_mod_exp_backup)(BIGNUM * r0, const BIGNUM * I, RSA * rsa,
+                                       BN_CTX * ctx);
```

Comment:
Shouldn't RSA be 'const RSA'? At least in RSA_get0_key() it's const ...

Reply:
No, the backup is fetched via this function: